src/encoded/static/components/regulome_search.js

```diff
@@ -597,11 +597,21 @@ export class RegulomeSearch extends React.Component {
         if (this.context.location_href.split('/thumbnail=')[1] === 'valis') {
             this.chooseThumbnail('valis');
         }
-
+        // the goal is to pick 1 each bigWig and bigBed file per experiment, with the following output types and replicate numbers for the different assays:
+        // DNase → peaks, read-depth normalized signal (rep1)
+        // ChIP → peaks and background as input for IDR, signal p-value (rep1,2) or rep1
+        // FAIRE → peaks, signal
+        // we start by collecting all files that satisfy these conditions
         if (this.state.genome === 'hg19') {
-            this.loadHg19ValisData();
+            const chipBaseQuery = `type=File&assembly=${this.state.genome}&file_format=bigBed&file_format=bigWig&assembly=hg19&preferred_default=true&sort=dataset&limit=all&${fieldsToSave}`;
+            const dnaseBaseQuery = `type=File&assembly=${this.state.genome}&file_format=bigBed&file_format=bigWig&assembly=hg19&preferred_default=true&sort=dataset&limit=all&${fieldsToSave}`;
+            const faireBaseQuery = `type=File&assembly=${this.state.genome}&file_format=bigBed&file_format=bigWig&assembly=hg19&preferred_default=true&sort=dataset&limit=all&${fieldsToSave}`;
+            this.loadValisData(chipBaseQuery, dnaseBaseQuery, faireBaseQuery);
```

Comment:
Sort of odd all three queries are composed independently, yet all three are identical. I realize this is from existing code though.